text/0028-override-value-operators.md

```diff
@@ -0,0 +1,65 @@
+- Start Date: 2023-10-05
```

Comment:
This should be today's date (of the meeting where the RFC was accepted), not the date you drafted it.